src/components/PageDefinitions.js

```diff
@@ -223,7 +229,13 @@ class PageDefinitions extends AbstractPageDefinitions {
 
   dropZone(child) {
     return (
-      <Dropzone disableClick onDrop={this.onDrop} style={{ position: 'relative' }}>
+      <Dropzone
+        accept="application/json"
```

Comment:
how will this work when we start to accept lots more types of files? Right now it is just json cause that was simple and easy but more generally we will be taking just about anything we can understand. It's ok for now this way but I'm interested to understand the future.

Note also that we need to be able to drop things other than files at some point. In particular, if someone drags a URL from a browser.

Reply:
This needs of course to be improved, there are already some opened issues about DnD and the ability to allow any kind of accettable data.
We would end with a specific utils class method that return the allowed file formats, and of course with a related method that would check the format of those files.
For DnD we need to understand if DropZone is able to manage them, or if we should use something different (see comments on #168 )